node/in_request.js

```diff
@@ -145,8 +146,13 @@ TChannelInRequest.prototype.handleFrame = function handleFrame(parts, isLast) {
 TChannelInRequest.prototype.emitError = function emitError(err) {
     var self = this;
 
-    self.err = err;
-    self.errorEvent.emit(self, err);
+    if (LiteError.isLiteError(err)) {
```

Comment:
We need a new method. `emitError` and `emitErrorFrame`. 

That's the cleanest way to do this.